samples/scaleout/distributor-upgrade/sample.md

```diff
@@ -21,6 +23,17 @@ This sample uses the same solution as the Version 5 [distributor sample](/sample
 
 ## Upgrade
 
+NServiceBus 6 does not have a distributor role. The distributor remains on NServiceBus 5.x, the workers can upgrade to 6.x.
+
+
+Warning: It is not recommended to use Outbox on the workers. Enabling Outbox can result in increasing the enlisted capacity of a worker when errors occur. The distributor model is originally designed to work with distributed transactions (MSDTC). Consider using [Sender Side Distribution](/transports/msmq/sender-side-distribution) instead. When enabling Outbox on a worker, explicitly set transaction mode to "Sends atomic with receive" as the Outbox sets the default transaction mode to "Receive Only".
```

Comment:
I think this section should be in the general distributor doco, not in the upgrade guide. If somebody has a working V5 solution with a distributor, he has these settings set correctly.